packages/api-client/src/v2/blocks/response-block/ResponseBlock.vue

```diff
@@ -177,7 +177,10 @@ defineExpose({
           :layout="layout"
           :totalPerformedRequests="totalPerformedRequests"
           @addRequest="
-            eventBus.emit('ui:open:command-palette', { action: 'addOperation' })
+            eventBus.emit('ui:open:command-palette', {
+              action: 'create-request',
+              payload: undefined,
```

Comment:
Just make the payload optional instead of having to pass undefined here like we did for the whole payload (see two lines below)
```suggestion
```